.github/workflows/api-test.yaml

```diff
@@ -42,6 +42,19 @@ jobs:
             ~/.gradle/wrapper
           key: v1-${{ runner.os }}-gradle-${{ hashfiles('**/gradle-wrapper.properties') }}-${{ hashFiles('**/*.gradle') }}
           restore-keys: v1-${{ runner.os }}-gradle-${{ hashfiles('**/gradle-wrapper.properties') }}
+      - name: Pull credentials
+        id: pull_credentials
+        run: |
+          # For security reasons, Broad prefers we read GHA secrets instead of reading from vault.
+          # this step does the equivalent of the pull-credentials.sh script.
+          # on local machines, the script fetches a SA from Vault.
+          # in GH actions, the SA key is stored in a GH repo secret.
+          # regardless of how it was fetched, tests and scripts expect these
+          # keys to be stored in rendered/
+          mkdir -p rendered/broad/
```

Comment:
Yes, we need credentials so that our tests can talk to BQ (e.g. to check if the dataset exists, to fetch data types, to compute display hints). This is a DataBiosphere repo, not a verily-src one, so the tests talk to Broad-owned infrastructure, not Verily-owned. Broad's approach to running tests against their infrastructure is to copy an SA key file into a GH repo secret, and then use those credentials in the GHA.